draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -200,6 +200,30 @@ Other techniques such as email or HTTP(S) based validation are out-of-scope.
 
 * `Random Token`: a random value that uniquely identifies the DNS domain control validation challenge, defined in {{random-token}}.
 
+# Purpose of Domain Control Validation {#purpose}
+
+Domain Control Validation (DCV) is a challenge-response protocol that allows one party (the User) to prove operational control of the contents of a domain to an Application Service Provider (ASP).  This proof applies at a single point in time; it does not provide evidence of continuing control prior to the challenge or after the record is published.
```

Comment:
Should we move the "(ASP)" up to the definitions above where we define Application Service Provider, or perhaps just expand ASP throughout?

Reply:
My suggestion is add the (ASP) to the definition, but for now do not use it (so we expand it in this pull request). 
My logic is have it in place so it can be used - rfc editor or someone may think it is a good idea